toffy/panel_utils.py

```diff
@@ -0,0 +1,106 @@
+import os
+import pandas as pd
+
+from pathlib import Path
+
+from ark.utils import io_utils
+
+
+def convert_panel(panel_path):
+    """
+    Converts the panel retrieved from ionpath into a necessary toffy format,
+    also adds necessary channels for analysis
+    Args:
+        panel_path (str): direct path to panel file
+    Returns:
+        panel (pd.DataFrame): detailing the Mass, Target, Start, and Stop values
+        also saves the above to file with name formatted "panel_name-toffy.csv"
+    """
+
+    # retrieve the panel name and directory it is contained in
+    panel_name = os.path.basename(panel_path).split('.')[0]
+    panel_dir = os.path.dirname(panel_path)
+
+    # read in the example_panel_file to have necessary additional targets
+    example_panel = pd.read_csv(os.path.join(Path(__file__).parent.parent, 'files',
```

Comment:
I think it's fine to define this as global variables in settings rather than reading in the example panel